suite.lisp

```diff
@@ -0,0 +1,77 @@
+(in-package #:org.shirakumo.trivial-benchmark)
+
+(defvar *benchmark-packages* (make-hash-table :test 'eq)
+  "A table mapping package objects (which are benchmark packages) to a list of fbound symbols.")
+
+(defun find-benchmark-package (package-designator)
+  "Find the benchmark package designated by PACKAGE-DESIGNATOR and return it, otherwise return NIL if it doesn't exist."
+  (let ((package-object (find-package package-designator)))
+    (if (or (null package-object)
+            (null (nth-value 1 (gethash package-object *benchmark-packages*))))
+        nil
+        package-object)))
```

Comment:
Why not just:
`(and (nth-value 1 (gethash package-object *benchmark-packages*)) package-object)`
Or in the very least
`(when (and package-object (nth-value 1 (gethash package-object *benchmark-packages*))) package-object)`
would be a lot clearer imo.